Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/execution/Context.java

```diff
@@ -644,6 +644,14 @@ public void setParameter(String libraryName, String name, Object value) {
         }
     }
 
+    public void setParameters(Library library, Map<String, Object> parameters) {
+        if (parameters != null) {
+            for (Map.Entry<String, Object> parameterValue : parameters.entrySet()) {
+                setParameter(null, parameterValue.getKey(), parameterValue.getValue());
```

Comment:
Just thinking : is setParameter(), libraryName always be null

Reply:
It won't always be null, but if it is, it means the parameter will resolve in any library. If it's not, (i.e. it's qualified), it will only resolve in that library.